tests/virtual_machines_test.go

```diff
@@ -127,6 +127,23 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				})
 			})
 
+			Context("and a simple vm is applied", func() {
+				It("should not remove the transactions timestamp annotation", func() {
+					vm := CreateVmObject(TestNamespace, []kubevirtv1.Interface{newInterface("br", "")}, []kubevirtv1.Network{newNetwork("br")})
+					vm, err := testClient.VirtClient.VirtualMachine(vm.Namespace).Create(context.TODO(), vm, metav1.CreateOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					Consistently(func() error {
+						vm, err = testClient.VirtClient.VirtualMachine(vm.Namespace).Get(context.TODO(), vm.Name, metav1.GetOptions{})
+						Expect(err).ToNot(HaveOccurred())
+						if val, exist := vm.Annotations[pool_manager.TransactionTimestampAnnotation]; !exist || val == "" {
+							return fmt.Errorf("vm %s should have a transaction timestamp annotation", vm.Name)
+						}
+						return nil
+					}).WithPolling(time.Second).WithTimeout(5 * time.Second).Should(Succeed())
```

Comment:
Use the proper gomega matches
```suggestion
					Consistently(func() (string, error) {
						vm, err = testClient.VirtClient.VirtualMachine(vm.Namespace).Get(context.TODO(), vm.Name, metav1.GetOptions{})
						if err != nil {
						   return "", err
						}
						return vm.Annotations[pool_manager.TransactionTimestampAnnotation], nil
					}).WithPolling(time.Second).WithTimeout(5 * time.Second).ShouldNot(BeEmpty(), "vm %s should have a transaction timestamp annotation", vm.Name)
```

Reply:
[Fixed](https://github.com/k8snetworkplumbingwg/kubemacpool/compare/d03888e12e3c9a5ecba4a5ee4e12607b41ae4935..a1d5389f7c6f1a25cc2d69856b47c28cf698763f)